webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts

```diff
@@ -174,10 +175,7 @@ export class ConnectionOptionsTabService extends Bootstrap {
     const driver = await this.dbDriverResource.load(state.config.driverId, ['includeProviderProperties']);
 
     state.config.authModelId = driver?.defaultAuthModel;
-
-    state.config.configurationType = driver?.configurationTypes.includes(DriverConfigurationType.Manual)
-      ? DriverConfigurationType.Manual
-      : DriverConfigurationType.Url;
+    state.config.configurationType = driver ? getDefaultConfigurationType(driver) : DriverConfigurationType.Manual;
```

Comment:
logic is a bit different, previously when `driver === undefined` we will use `DriverConfigurationType.Url`
now it's `DriverConfigurationType.Manual`.
I will recommend changing the `getDefaultConfigurationType` function to accept driver | undefined and determine the type completely within this function.